### ISSUE TITLE
Reset button is disappearing after control rebuild

### ISSUE THREAD

**Issue by astrikov-d** [davidstutz/bootstrap-multiselect#987]
Hello!
First of all, thank you for your great work on this plugin, it helps me a lot.

Not long time ago you've added support of "reset" button: https://github.com/davidstutz/bootstrap-multiselect/pull/678
But this button is disappearing after multiselect rebuit.

I guess, `this.buildReset();` is missed inside `rebuild` function:

There is:

```
...
this.buildSelectAll();
this.buildDropdownOptions();
this.buildFilter();

this.updateButtonText();
this.updateSelectAll(true);
...
```

I guess, it should be:

```
...
this.buildSelectAll();
this.buildDropdownOptions();
this.buildFilter();
this.buildReset();

this.updateButtonText();
this.updateSelectAll(true);
...
```

May be it's missed in some other places too.

**Comment by mrfisher8432** [comment]
This is correct. I added the line and it works. However, I would propose you put the code in this order:

`
this.buildSelectAll();
this.buildDropdownOptions();
this.buildReset();
this.buildFilter();
`

**Comment by DaveSenn** [comment]
Would love to see this change beeing merged and published...
